File: python/activity_1/pseudo_code.py
# AREA & CIRCUMFERENCE CALCULATOR

print("\n## AREA & CIRCUMFERENCE CALCULATOR")
def get_validate_number(prompt):
    input_number = input(prompt)
    if not (input_number.isnumeric()):
        print("\n## Error: Enter a valid number\n")
        quit()

    return float(input_number)

pi = 3.142
radius = get_validate_number("Enter the circle radius(meters): ")
circumference = 2 * pi * radius
area = pi * (radius**2)

print("Results:\nCircumference: %.3fm\nArea: %.3fm" % (circumference, area))

File: php/activity_1/pseudo_code.php
<?php
# AREA & CIRCUMFERENCE CALCULATOR
function get_validate_number($prompt) {
    echo $prompt;
    $input = trim(fgets(STDIN));

    if (!is_numeric($input)) {
        echo "\n## Error: Enter a valid number\n";
        exit();
    }
    return floatval($input);
}

$pi = 3.142;
$radius = get_validate_number("Enter the circle radius: ");
$circumference = 2 * $pi * $radius;
$area = $pi * $radius**2;

echo "Results:\nCircumference: $circumference\nArea: $area ";